src/massDistribution/Ferriere.cpp

```diff
@@ -222,12 +220,11 @@ double Ferriere::getNucleonDensity(const Vector3d &position) const{
 		n+= 2*getH2Density(position);
 	}
 
-	// check if any density is activ and give warning if not
-	if(isforHI||isforHII||isforH2 == false){
+	// check if all densities is active and raise warning if not
+	if((isforHI & isforHII & isforH2) == false){
```

Comment:
You probably mean &&  instead of &

Reply:
this is the way like it is introduced  #336 by @lukasmerten. 
I have just copied the code to resolve a merge conflict.

Reply:
& is a bitwise and while && is logical and. Bitwise and is not what is wanted here. For boolleans the result is the same, but it is still confusing.

Reply:
okay. I have changed it at this part, but I think this done in some other contexts to.